admin_interface/admin.py

```diff
@@ -15,34 +15,34 @@ class ThemeAdmin(admin.ModelAdmin):
             'classes': ('wide', ),
             'fields': ('name', 'active', )
         }),
-        ('Enviroment', {
+        (_('Enviroment'), {
```

Comment:
Could take the opportunity to fix the typo here :)

Reply:
Done!